### DISCUSSION TITLE
How to?

### DISCUSSION THREAD

**Discussion by xMvdl-ui** [fabiusBile/Spotify4BigSur#13] [General]
Maybe I am dumb, but how do you install this?
In this Github I don't see a SpotifyMain.app ?

**Comment by exastone** [comment]
on the title page of the project theres a "Release" section to the right-hand side;
click on that then download _SpotifyMain.dmg_, then install the image (the .dmg). After this there should be a prompt to drag and drop SpotifyMain to the applications folder. Enjoy!
